api/kubernetes/deploy_via_helm-dev.sh

```diff
@@ -218,3 +218,15 @@ helm upgrade --install "${COSMOTECH_API_RELEASE_NAME}" "${HELM_CHARTS_BASE_PATH}
     --set config.csm.platform.argo.workflows.namespace="${NAMESPACE}" \
     --set podAnnotations."com\.cosmotech/deployed-at-timestamp"="\"$(date +%s)\"" \
     "${@:4}"
+
```

Comment:
We may need to put this install script outside deploy_via_helm if it stays in a common monitoring namespace (multitenant deployment).
For now maybe create a new namespace with ${NAMESPACE}-monitoring to have it scoped to only 1 tenant (multi tenant will use namespace).
Or let it like that if you do not want to anticipate.

Reply:
Set to ${NAMESPACE}-monitoring